examples/sensors-viewer/src/components/FlashHomeModal.jsx

```diff
@@ -0,0 +1,60 @@
+import React, {Component} from 'react';
+import {Button, Col, Form, Modal} from "react-bootstrap";
+import {setAuthToken, setRealmName} from "../controller/storageService";
+
+class FlashHomeModal extends Component {
```

Comment:
FlashHomeModal reminds me of flash messages, like alerts, i think we should rename this to something related to its specific use, like CredentialsModal, RealmConfigModal or something along those lines